File: BOR2/apps/api/internal/domain/takeoff.go
package domain

import "time"

// TakeoffWork maps to the takeoff_works table.
type TakeoffWork struct {
	ID                   string     `json:"id"`
	Project              string     `json:"project"`
	DataSolicitacao      *time.Time `json:"dataSolicitacao,omitempty"`
	DataInicio           *time.Time `json:"dataInicio,omitempty"`
	DataEstimadaEntrega  *time.Time `json:"dataEstimadaEntrega,omitempty"`
	EntregaReal          *time.Time `json:"entregaReal,omitempty"`
	Description          string     `json:"description"`
	DocLinks             string     `json:"docLinks"`
	ModeloDaCasa         string     `json:"modeloDaCasa"`
	StageDwg             string     `json:"stageDwg"`
	StageMitek3d         string     `json:"stageMitek3d"`
	StageMaterialsList   string     `json:"stageMaterialsList"`
	StagePanelDivision   string     `json:"stagePanelDivision"`
	StageValidation      string     `json:"stageValidation"`
	StageCutList         string     `json:"stageCutList"`
	StageProduction      string     `json:"stageProduction"`
	StageDelivery        string     `json:"stageDelivery"`
	StageAssembly        string     `json:"stageAssembly"`
	CreatedAt            time.Time  `json:"createdAt"`
}

type TakeoffWorkFilters struct {
	Project string
}

File: BOR1/frontend/src/types/takeoff.ts
export interface TakeoffRow {
  id: string;
  project: string;
  data_solicitacao: string;
  data_inicio: string;
  data_estimada_entrega: string;
  entrega_real: string;
  description: string;
  doc_links: string;
  modelo_da_casa: string;
  opcionais_da_casa: string;
  arquivo_dwg: string;
  plano_estrutural: string;
  adequacao_dwg: string;
  importacao_dwg_mitek: string;
  execucao_3d_mitek: string;
  lista_materiais_excel: string;
  dividir_3d_paineis: string;
  validacao_projeto_takeoff: string;
  created_at?: string;
} 